fetch_raw_data.py

```diff
@@ -137,6 +141,54 @@ def fetch_from_gcloud_bucket(google_cloud_credentials_file_path, raw_data_dir, g
                 google_cloud_credentials_file_path, blob_url, traced_runs_output_file)
 
 
+def fetch_from_shaqadoon_csv(user, google_cloud_credentials_file_path, raw_data_dir, phone_number_uuid_table,
+                             shaqadoon_csv_source):
+    log.info("Fetching data from a Shaqadoon CSV...")
+    for blob_url in shaqadoon_csv_source.activation_flow_urls + shaqadoon_csv_source.survey_flow_urls:
+        traced_runs_output_path = f"{raw_data_dir}/{blob_url.split('/')[-1].split('.')[0]}.jsonl"
```

Comment:
An example in a comment would help legibility here

Reply:
Added a comment and moved the split code to a separate variable to also help with legibility